packages/cardano-services/src/TxSubmit/rabbitmq/TxSubmitWorker.ts

```diff
@@ -294,7 +294,7 @@ export class TxSubmitWorker extends EventEmitter {
       const txBody = new Uint8Array(content);
 
       // Register the handling of current transaction
-      txId = cmlUtil.deserializeTx(txBody).id;
+      txId = (await cmlUtil.deserializeTx(txBody)).id;
```

Comment:
Just noticed that we have a more efficient way to do this:
```suggestion
      txId = TransactionId.fromTxBodyCbor(TxBodyCBOR.fromTxCBOR(txBody))
```
Maybe we can also update those utils to (at least partially) use new serialization?